pmd-cpp/src/test/java/net/sourceforge/pmd/cpd/CPPTokenizerContinuationTest.java

```diff
@@ -25,35 +24,32 @@
     public void parseWithContinuation() throws Exception {
         String code = load("cpp_with_continuation.cpp");
         Tokens tokens = parse(code);
-        if (tokens.size() < 53) {
+        if (tokens.size() < 52) {
             printTokens(tokens);
-            fail("Not enough tokens - probably parsing error");
+            fail("Not enough tokens - probably parsing error. Tokens: " + tokens.size());
         }
 
         assertEquals("static", findByLine(8, tokens).get(0).toString());
         assertEquals("int", findByLine(8, tokens).get(1).toString());
 
-        // Note: the token should be "ab", but since we skip "\\\n" between a and b,
-        // we end up with two tokens. -> the tokens are not joined.
-        // This could lead to false negatives in duplication detection.
-        // However, this is only a problem, if the continuation character is added *within*
-        // a token and not at token boundaries.
+        // special case, if the continuation is *within* a token
         // see also test #testContinuationIntraToken
-        //assertEquals("ab", findByLine(8, tokens).get(2).toString());
+        assertEquals("ab", findByLine(8, tokens).get(2).toString());
```

Comment:
maybe add an assert checking the token's beingLine / endLine are properly filled?